registrations/tasks.py

```diff
@@ -198,7 +231,7 @@ def create_subscriptionrequests(self, registration):
             "identity": registration.registrant_id,
             "messageset": msgset_id,
             "next_sequence_number": next_sequence_number,
-            "lang": registration.data["language"],
+            "lang": "eng_ZA",  # NurseConnect is currently only in english
```

Comment:
Not sure this is something worth putting in code in here. The only reason for this is lack of translated content. I think we should move this constraint to the jsbox app only.